.github/workflows/push.yml

```diff
@@ -0,0 +1,25 @@
+name: Push to S3
+
+on:
+  push:
+    branches:
+    - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+    - uses: actions/checkout@v4
+
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v4
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: ${{ secrets.AWS_REGION }}
+
+    - name: Sync files to S3 bucket and Invalidate 
+      run: |
+        aws s3 sync ./ s3://din.global --delete --exclude '.git*/*'
```

Comment:
this is the actual name of the bucket?

din.global??

Reply:
yes. `din.global` is bucket name.